e2e_tests/tests/cluster/test_checkpoints.py

```diff
@@ -75,25 +76,40 @@ def run_gc_checkpoints_test(checkpoint_storage: Dict[str, str]) -> None:
     # Check that the actual checkpoint storage (for shared_fs) reflects the
     # deletions. We want to wait for the GC containers to exit, so check
     # repeatedly with a timeout.
+
     max_checks = 30
     for i in range(max_checks):
         time.sleep(1)
         try:
+            storage_states = []
             for config, checkpoints in all_checkpoints:
                 checkpoint_config = config["checkpoint_storage"]
 
                 storage_manager = storage.build(checkpoint_config, container_path=None)
+                storage_state = {}
                 for checkpoint in checkpoints:
                     storage_id = checkpoint["uuid"]
+                    # Trying again
```

Comment:
What does this comment mean?

Reply:
Not sure how this got in there, removing it.